nuxeo-drive-core/src/main/resources/OSGI-INF/nuxeodrive-adapter-service.xml

```diff
@@ -56,6 +56,8 @@
           </fileSystemItemFactory>
          </extension>
       </code>
+      Please note that `versioningDelay` and `versioningOption` parameter have been deprecated since 9.1 and are not
+      used anymore as automatic versioning is now handled at versioning service level.
 
       Example of a FileSystemItem factory for documents with the Picture facet:
       <code>
```

Comment:
We should remove the versioning parameters at:
- https://github.com/nuxeo/nuxeo-drive-server/blob/feature-NXP-21275-new-versioning-system/nuxeo-drive-core/src/test/resources/OSGI-INF/test-nuxeodrive-adapter-service-contrib.xml#L14
- https://github.com/nuxeo/nuxeo-drive-server/blob/feature-NXP-21275-new-versioning-system/nuxeo-drive-core/src/test/resources/OSGI-INF/test-nuxeodrive-adapter-service-contrib-override.xml#L19